lib/install/stimulus.rb

```diff
@@ -0,0 +1,12 @@
+say "Appending Stimulus setup code to #{Webpacker.config.source_entry_path}/application.js"
+inject_into_file "#{Webpacker.config.source_entry_path}/application.js", after: "console.log('Hello World from Webpacker')" do
```

Comment:
Injecting after `console.log('Hello World from Webpacker')` feels kind of brittle. Maybe just append to the end?